modulus/models/registry.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Can this file just be in `modulus/registry/model_registry.py` and not have the last line there?

Then we can have a `modulus/registry/__init__.py` be
```python
from model_registry import ModelRegistry
```

A user would use it as 
```python
from modulus.models import Module
from modulus.registry import ModelRegistry

class CustomModel(Module):
    ...

# Since singleton class, don't need to keep around an instantiated version of this class
ModelRegistry().register(CustomModel, "custom_model")
```

Reply:
Changed it to this.